ext/iodine/iodine_http.c

```diff
@@ -365,7 +365,11 @@ static inline VALUE copy2env(iodine_http_request_handle_s *handle) {
   if (*pos == 0) {
     rb_hash_aset(env, SERVER_NAME,
                  rb_enc_str_new(tmp.data, tmp.len, IodineBinaryEncoding));
-    rb_hash_aset(env, SERVER_PORT, QUERY_ESTRING);
+    if (RSTRING_LEN(QUERY_ESTRING) != 0) {
```

Comment:
This is my fault for having poorly named variables, but... the `ESTRING` in `QUERY_ESTRING` stands for an **Empty** String, its length is always zero and it's used almost everywhere I needed an empty frozen string (to avoid creating multiple objects)... so there's really no need for this `if` statement.